### ISSUE TITLE
Paging logic seems  flawed

### ISSUE THREAD

**Issue by manosbatsis** [MahirMahdi/refine-jsonapi#1]
I believe your paging logic is intended for `page[offset]` and `page[limit]` URL params, but you use it to create `page[number]` and `page[size]` instead.

**Comment by MahirMahdi** [comment]
Yes I used them because the naming conventions are common in refine. If it causes any issues let me know, I'll update it.
